src/foam/mlang/mlang.js

```diff
@@ -4501,6 +4501,15 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      class: 'String',
+      name: 'id',
```

Comment:
Why is this needed?

Reply:
I suspect it might not be needed https://github.com/nanoPayinc/NANOPAY/pull/14146#discussion_r818604591.

Reply:
reverted in final version